package.json

```diff
@@ -100,6 +100,14 @@
     "test": "npm-run-all --serial test:*",
     "watch": "npm run test:jest -- --watch"
   },
+  "eslintConfig": {
```

Comment:
It's set [`eslint-config-stylelint`](https://github.com/stylelint/eslint-config-stylelint/blob/master/eslintrc.json#L8)